controllers/releaseplan/controller.go

```diff
@@ -60,15 +63,26 @@ func (c *Controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	return controller.ReconcileHandler([]controller.Operation{
 		adapter.EnsureOwnerReferenceIsSet,
+		adapter.EnsureMatchingInformationIsSet,
```

Comment:
I comment here because I cannot comment in the line I need. This controller needs rback for RPAs as well.

Reply:
fixed